tgui/packages/tgui/interfaces/TurretControl.js

```diff
@@ -0,0 +1,72 @@
+import { Window } from '../layouts';
+import { useBackend, useLocalState } from '../backend';
+import { Button, Section, Flex, Box, Stack } from '../components';
+
+export const TurretControl = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    enabled,
+    lethal,
+    emagged,
+    area,
+  } = data;
+
+  const set_lethal = (value) => {
+    if (value === !!lethal) {
+      return;
+    }
+    act("setLethal", { "lethal": value });
+  };
+  const set_enabled = (value) => {
+    if (value === !!enabled) {
+      return;
+    }
+    act("setEnabled", { "enabled": value });
+  };
+  return (
+    <Window
+      title={emagged ? "FATAL ERROR" : "Turret control (" + area + ")"}
+      theme={emagged ? 'syndicate' : 'ntos'}
+      width={400}
+      height={160}
+    >
+      <Window.Content align="center">
+        <br />
+        {!emagged && (
+          <Box>
```

Comment:
I think this `Box` is just being used as a structural container of the two `Section` components rather than any visual thing? If so, replace with fragments (`<>` and `</>`).